scipy/stats/_discrete_distns.py

```diff
@@ -1049,6 +1066,89 @@ def _munp(self, n, a):
 zipf = zipf_gen(a=1, name='zipf', longname='A Zipf')
 
 
+def _gen_harmonic(n, a=1):
+    """Generalized harmonic number"""
+    # See https://en.wikipedia.org/wiki/Harmonic_number; search for "hurwitz"
+    # todo - make this work for a < 1 (can't use zeta)
+    return zeta(a, 1) - zeta(a, n+1)
```

Comment:
How should I implement this for a <=1 ? I was thinking a naive (just summing the harmonic series) in a ufunc. I could try to do something fancy from within Python that saves computation (e.g. for each distinct `a`, generate the harmonic series up to the maximum of the `n`s, perform a cumulative sum for the `n`s that are needed) but I'm not sure it's worth the trouble.